phobos/PackageMap.md

```diff
@@ -1,19 +1,69 @@
 ## Phobos 3 Package Map
 
+`based` modules duplicated in `phobos` will contain a mix of wrapped `based` functions, new fuctions, and public imports of `based` symbols, depending on what ends up being the most effective solution.
+
+Packages/Modules with an asterisk are new.
+
 ### Open Questions
 
 1. `std.container`: Does it belong in `sys` or it's own root? This likely hinges on the status of [DIP1000](https://github.com/dlang/DIPs/blob/master/DIPs/other/DIP1000.md) and whether or not it needs a ground up rewrite.
 2. `std.concurrency`: It is possible that not all platforms support Concurrency mechanics. Should `std.concurrency` be required in the core roots or moved to a non-core root?
 3. `std.parallelism`: It is possible that not all platforms support Task Parallelism mechanics. Should `std.parallelism` be required in the core roots or moved to a non-core root?
-4. `std.digest`: Cryptography routines have unique requirements that would be best served by providing them in their own root. But in all cases providing bespoke implementations of cryptography routines is never recommended. Third-party trusted implementations should be used instead. This could be from OpenSSL/LibreSSL (POSIX), SChannel/BCrypt (Windows), or CryptoKit (MacOS/IOS). It is acceptable to provide implementations of Non-Cryptographic routines such as CRC and Murmur. This would entail removing all the digests except CRC and Murmur from this package and building a separate cryptography package.
-5. Which, if any, modules should be removed? This could be either a permanent removal or pending replacement with ground-up rewrites.
+4.  Which, if any, modules should be removed? This could be either a permanent removal or pending replacement with ground-up rewrites.
+
+### Closed Questions
+1. `std.digest`: Only non-cryptographic digests will be kept. SHA2/3 will be available in `phobos.sys.hash` using the system provided API.
+2. Removed modules:
+    1. `std.json` (Replaced with [JSONIOPipe](https://github.com/schveiguy/jsoniopipe))
+    2. `std.getopt`
+    3. `std.logger`
+    4. `std.experimental`
+    5. `std.digest` (only: `hmac`/`md`/`ripemd`/`sha`)
+3. `std.typecons` will be split into distinct modules.
 
-### Proposed Package Structure (Existing Modules Only)
+### Proposed Package Structure
+
+***This map is not final!***
 
 ```
 core.*
 etc.*
 std.*
+platform
+  | freebsd
+  | linux
+  | macos
+  | stdc
+  | win
+based
```

Comment:
Some discussion on discord about whether the expectation was for `based` or for `base` and BaseD was just a nickname.

I sorta like `base` better, as `based` is a bit cute. We also don't say `phobosd` or `platformd`.